packages/app-desktop/gui/MasterPasswordDialog/Dialog.tsx

```
@@ -103,6 +103,9 @@ export default function(props: Props) {
 	}, []);
 
 	// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
+	/**
```

ESLint

[error] Expected multiple line comments instead of a block comment.

(multiline-comment-style)